Library/Formula/jbig2dec.rb

```diff
@@ -12,9 +10,15 @@ class Jbig2dec < Formula
     sha1 "cbb4dfe055be243427210d03304c20760fb00bd7" => :lion
   end
 
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+
   def install
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
-    system "make install"
+    system "autoreconf", "-fvi" # error: cannot find install-sh
+    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}",
+                          "--disable-silent-rules", "--without-libpng"
```

Comment:
It looks for a header that Homebrew's linpng doesn't ship with for some yet undiscovered reason and when it can't find that header compile breaks.

Sent from my iPhone

> On 27 Mar 2015, at 09:29, Xu Cheng <notifications@github.com> wrote:
> 
> In Library/Formula/jbig2dec.rb:
> 
> >    def install
> > -    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
> > -    system "make install"
> > +    system "autoreconf", "-fvi" # error: cannot find install-sh
> > +    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}",
> > +                          "--disable-silent-rules", "--without-libpng"
> Why --without-libpng ?
> 
> —
> Reply to this email directly or view it on GitHub.
>